arducate/src/components/Toolbar.js

```diff
@@ -85,9 +146,25 @@ const Toolbar = () => {
         >
           Publish
         </Button>
+
+        {/* Share Button */}
+        <Button
+          variant="outline"
+          className="navbar-button"
+          onClick={handleShare}
+        >
+          Share
+        </Button>
+
+        {/* Popup Notification */}
+        {showPopup && (
+          <div className="absolute top-12 right-0 bg-green-500 text-white p-2 rounded">
+            Link copied to clipboard!
```

Comment:
can we also show the link here, it makes it easier incase theres a problem or someone just wants to read it out